index.html

```diff
@@ -2,14 +2,21 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <title>Curriculum</title>
+    <title>Pyonguk Agustin | Portfolio</title>
     <link rel="icon" href="icons/portfolio.png">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" type="text/css" href="phone_style.css">
+    <link rel="stylesheet" type="text/css" href="desktop_style.css">
   </head>
 
   <body>
```

Comment:
- [ ] [OPTIONAL] Kindly let's remove the `overflow-x`, because it is creating a bad UX, you can use `overflow-x: hidden;` to the body 👍